app/views/mentee_profiles/show.html.slim

```diff
@@ -14,15 +14,14 @@
         = @user.program_country
         a.left10 = @user.timezone
   h3 Project
-  span.project-title Lorem ipsum
+  span.project-title = @project.title
   p
-    | Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
-    | tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
-    | quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-    | consequat.
-  p Odio nostrum nisi vel enim veritatis, facere sunt quo sequi vitae vero suscipit, consequatur cupiditate tenetur maxime perspiciatis neque provident hic ad veniam. Nostrum consequuntur tempore, saepe provident atque accusantium.
+     |Languages used
+     = @project.language
+     br
+     = @project.description
```

Comment:
the project description is a paragraph, if its long, then it should break the paragraph into two and then display it ? or everything is automatically handled :P